catch (Exception e) 
    { 
      e.printStackTrace(); 
    } 
 
    try 
    { 
